src/azul/service/elasticsearch_service.py

```diff
@@ -96,6 +96,9 @@ def _translate_filters(self, filters: Filters, field_mapping: JSON) -> MutableFi
             assert isinstance(value, dict)
             assert isinstance(one(value.values()), list)
             field_type = self.field_type(tuple(key.split('.')))
+            if key == 'contents.donors.organism_age':  # Corresponds to value of 'organismAge' facet
+                # Special handling required; https://github.com/DataBiosphere/azul/issues/1747#issuecomment-642247817
```

Comment:
That comment is a shorthand not that makes little sense in isolation. Don't link to it but instead describe the intent here explicitly.